a.css

```diff
@@ -1,8 +1,9 @@
+
 .body {
```

Comment:
- [ ] It would be better to remove the margins from the top, right, and left from your left to improve your UI design.

<img width="156" alt="image" src="https://user-images.githubusercontent.com/77190203/164862009-fb7f9e01-2ca5-4950-bf1c-d4c2cd4805cf.png">